README.md

```diff
@@ -31,37 +36,255 @@ npm install @uploadcare/upload-client --save
 
 ## Usage
 
+### High Level API
```

Comment:
We never mention any High Level API before this point; logically, this requires an update of the library description.

Reply:
This is API for the library, not about Upload API